mantidproject/mantid
# Windows admin system wide installation

**Description of work.**
This PR enables a system wide installation of Mantid on Windows machines when running as the administrator. 

A small change has also been made to the `create_package.sh` file so that the nsis log will be in the same location as the package when building your own package.

**To test:**
1. Download the Windows package built from this branch - https://builds.mantidproject.org/job/build_packages_from_branch/23/ 
2. Right click the package and select `Run as administrator`
3. Use your 03 account credentials to run it as administrator
4. Once it is all installed go to `C:/Users/Public/Public Desktop` and check that there is a shortcut for Mantid that has been added as a result of this install.


Fixes #33736 


*This does not require release notes* because **it is a regression caused by package changes in this release**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
